### ISSUE TITLE
Make the deadline more visible

### ISSUE THREAD

**Issue by fbidu** [luanfonceca/speakerfight#148]
I just missed a deadline because, despite having browsed the event's page more then 10 times and even voted for the proposals, I just didn't see the deadline!

Just today, when the deadline was due, I saw where it was. I don't know if it affects more people or if it is just the case that I need a new pair of glasses haahah. Be that as it may, I think it won't harm to make the deadline more visible.

**Comment by vmesel** [comment]
It will be done! Probably using CSS styling and other things!

**Comment by luanfonceca** [comment]
@fbidu in that cases a good thing to do is to talk with the organizer of the event, in some cases they will let you submit a propose anyway.

But we should think in a better way to expose this dates.

**Comment by fbidu** [comment]
Yeah I tried that... Unfortunately, the proposals where already voted when I noticed. It was about Python Brasil 2016, I ended up submitting my talk as a workshop and got approved :smile:
